packages/now-client/src/utils/index.ts

```diff
@@ -81,17 +81,24 @@ export async function buildFileTree(
   path: string | string[],
   isDirectory: boolean,
   debug: Debug
-): Promise<string[]> {
+): Promise<{ fileList: string[]; ignoreList: string[] }> {
+  const ignoreList: string[] = [];
   let fileList: string[];
-  let { ig } = await getVercelIgnore(path);
+  let { ig, ignores } = await getVercelIgnore(path);
 
-  debug(`Found ${ig.ignores.length} rules in .vercelignore`);
+  debug(`Found ${ignores.length} rules in .vercelignore`);
   debug('Building file tree...');
 
   if (isDirectory && !Array.isArray(path)) {
     // Directory path
-    const cwd = process.cwd();
-    const ignores = (absPath: string) => ig.ignores(relative(cwd, absPath));
+    const ignores = (absPath: string) => {
+      const rel = relative(path, absPath);
```

Comment:
Was this a bug before? It was previously relative to the `cwd`?

Reply:
Ya, seemed like a bug to me.